packages/world/src/utils/InventoryUtils.sol

```diff
@@ -121,9 +119,9 @@ library InventoryUtils {
     return slot;
   }
 
-  function addObject(EntityId owner, ObjectTypeId objectType, uint128 amount) public {
+  function addObject(EntityId owner, ObjectType objectType, uint128 amount) public {
     require(amount > 0, "Amount must be greater than 0");
-    uint16 stackable = ObjectTypeMetadata._getStackable(objectType);
+    uint16 stackable = objectType.getStackable();
     require(stackable > 0, "Object type cannot be added to inventory");
```

Comment:
re previous comment about stackable, this would never get triggered right

Reply:
good point!